splunk-otel-android/src/main/java/com/splunk/rum/FragmentTracer.java

```diff
@@ -26,12 +26,15 @@
     static final AttributeKey<String> FRAGMENT_NAME_KEY = AttributeKey.stringKey("fragmentName");
 
     private final String fragmentName;
+    private final String customScreenName;
```

Comment:
rather than saving the customScreenName only when it's "custom", what if you just saved the "screenName" and made it either the fragmentName or the customScreenName value, as appropriate. Then you can leave off the ternaries when using it.

Reply:
This did cross my mind....I'll give it a shot.